delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchViewValue.scala

```diff
@@ -33,14 +41,14 @@ object ElasticSearchViewValue {
     * @param permission        the permission required for querying this view
     */
   final case class IndexingElasticSearchViewValue(
-      resourceSchemas: Set[Iri],
-      resourceTypes: Set[Iri],
-      resourceTag: Option[Label],
-      sourceAsText: Boolean,
-      includeMetadata: Boolean,
-      includeDeprecated: Boolean,
+      resourceSchemas: Set[Iri] = Set.empty,
+      resourceTypes: Set[Iri] = Set.empty,
+      resourceTag: Option[Label] = None,
+      sourceAsText: Boolean = true,
+      includeMetadata: Boolean = true,
+      includeDeprecated: Boolean = true,
       mapping: Json,
-      permission: Permission
+      permission: Permission = Permission.unsafe("views/query")
```

Comment:
The default value should either come from config or be defined as a constant somewhere.

Reply:
We don't have them as configuration values (as we don't have them for anything else. But I agree it should be a constant somewhere, for visibility